client/my-sites/media-library/test/data-source.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * @format
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { noop } from 'lodash';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import MediaLibraryDataSource from 'my-sites/media-library/data-source';
+import { createReduxStore } from 'state';
+
+// we need to check the correct children are rendered, so this mocks the
+// PopoverMenu component with one that simply renders the children
+jest.mock( 'components/popover/menu', () => {
+	return props => <div>{ props.children }</div>;
```

Comment:
Ah nice, it just needed another layer.

Reply:
Yeah, this had me really confused at first, thinking I'd missed a(nother) breaking change in React 16! But no, it was just how require/import works behind the scenes :)